data/src/main/java/co/nimblehq/compose/crypto/data/repository/GlobalRepositoryImpl.kt

```diff
@@ -0,0 +1,53 @@
+package co.nimblehq.compose.crypto.data.repository
+
+import android.content.Context
+import android.net.*
+import co.nimblehq.compose.crypto.domain.repository.GlobalRepository
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.MutableStateFlow
+
+class GlobalRepositoryImpl(
```

Comment:
Just a minor on the repository name, global shows the implicit on the data provided by the repository, for example: UserRepository would provide user data, or actions related to the user and so on. `GlobalStateRepository` would make more sense. To avoid the time-consuming of rebasing the PRs in the chain, you could update the last opening PR after merging this PR.